File: PLP_python/Aula03_ListaDeExercicios(15)/Q10_Calculator.py
class Q10_Calculator:
    def sum_two(self, a: int, b: int) -> int:
        """
        Sums two numbers.
        """
        return a + b

    def sum_three(self, a: int, b: int, c: int) -> int:
        """
        Sums three numbers.
        """
        return a + b + c

if __name__ == "__main__":
    calculator = Q10_Calculator()

    result1 = calculator.sum_two(10, 20)
    print("Sum of two numbers:", result1)

    result2 = calculator.sum_three(10, 20, 30)
    print("Sum of three numbers:", result2)

File: PLP_Java/src/Aula03_ListaDeExercicio/Q10_Calculator.java
package Aula03_ListaDeExercicio;

public class Q10_Calculator {
    public int sum(int a, int b) {
        return a + b;
    }

    public int sum(int a, int b, int c) {
        return a + b + c;
    }

    public static void main(String[] args) {
        Q10_Calculator calculator = new Q10_Calculator();

        int result1 = calculator.sum(10, 20);
        System.out.println("Soma de dois números: " + result1);

        int result2 = calculator.sum(10, 20, 30);
        System.out.println("Soma de três números: " + result2);
    }
}
